vision of the modpack/breakdown/general.md

```diff
@@ -0,0 +1,121 @@
+
+Should tedium be accepted/permitted for situations “outside of normal gameplay” - for example, is tedium associated with a no-rocket-run acceptable?
+</div>
+
+# 7. What is QoL?
+<div align="justify">
+Quality of Life is often using the pack code to automate “tedium” away - for example before VisualProspecting almost everyone would manually waypoint ore locations - something that is handled by VisualProspecting.
+
+Tools like NEI are an important part of QoL as it helps you find what you need.
+
+Additional QoL items include the questbook which lets you know about new things that can help your playthrough, along with UI improvements that reveal functionality.
+
+Some QoL should be available immediately at all times (Questbook, NEI, Journeymap, nice and functional GUIs, etc.).
+
+But there are other types of improvements and items sometimes labeled 'QoL' that simplify gameplay and bypass game mechanics (hatches and busses that integrate AE with GregTech, advanced ore scanners, automatic mining, mobility like flight/teleportation, etc.). Usually these should not be available immediately but should be seen as progressive upgrades to simplify things after the player has already done something dozens of times. These then also can be placed in the progression as nice rewards for unlocking a tier or making a complicated new machine (First EBF/Assembling Line/DTPF) or even promote the use of a otherwise optional mod.
+</div>
+
+# 8. What's our policy to add new mods?
+<div align="justify">
+All new mods must be FOSS (Free and Open Source Software) with no exceptions.
+Large mods must not only be “thematically appropriate” but also have a champion willing to work with the team both on development and balance, to fix issues and integrate the mod into the pack.
+Unless (and perhaps even) if actively under development, a fork shall be created in our repo.
+</div>
+
+# 9. What is the deprecation policy on removed or changed machines and recipes?
+<div align="justify">
+Many if not most players don’t follow GitHub development closely, and can be surprised by changes such as new lines or removed machines. This can result in already existing lines having to be revisited, which can be tedious.
+
+For removal of machines the process should be to first only remove the recipe and add a deprecation warning to the tooltip. And to only fully remove them at least one stable release after that.
+
+For recipe changes or removals, something like that is not feasible.
+
+However, in both cases the changes should be listed on the Breaking Changes page on the wiki (https://gtnh.miraheze.org/wiki/Upcoming_Features#Breaking_Changes). One should also check if there is an associated quest and change that as needed.
+</div>
+
+# 10. Miscellaneous Development Principles
+- Avoid small/tiny dusts if possible - percentage chance of full dusts instead. (See https://colab.research.google.com/drive/1hItspIDUBAAnNW4zJ1rMesAEiRytNWLh for some math on how to do that)
```

Comment:
`(See [this page](https://colab.research.google.com/drive/1hItspIDUBAAnNW4zJ1rMesAEiRytNWLh) for some math on how to do that)`